### DISCUSSION TITLE
UE 5.1 version

### DISCUSSION THREAD

**Discussion by Syntax3rror404** [Sixze/ALS-Refactored#207] [Q&A] [answered]
I wonder if a update of ALS ref to 5.1 planned because 5.1 has some cool new stuff and most chaos bugs fixed

**Comment by sommerlost** [comment, marked as answer]
The plugin currently does in fact work for 5.1 - You just need to to regenerate the  visual studio project, open up the .uplugin file for ALS and change the version number. Build the project source again and everything will be good to go.
